modules/nw-rfhe-subscribing-redfish-events.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+:_content-type: PROCEDURE
+[id="nw-rfhe-troubleshooting_{context}"]
+= Subscribing to Redfish Events
```

Comment:
Again, there are probably prereqs here, and it would be good to structure this as a formal procedure.